app/src/main/java/com/rentminder/SaveButton.kt

```diff
@@ -86,7 +77,7 @@ fun SaveRemindButton(enabled: Boolean, inputEdited: MutableState<Boolean>) {
 @Composable
 @Preview
 private fun SaveRemindButtonPreview() {
-    val enabled = false;
```

Comment:
Removed semicolon since the best practice is to omit them.

https://kotlinlang.org/docs/coding-conventions.html#semicolons